.github/workflows/tests.yml

```diff
@@ -0,0 +1,40 @@
+
+name: tests
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  linux_tests:
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: true
+      matrix:
+        php: ['7.1', '7.2', '7.3', '7.4']
+        stability: [prefer-lowest, prefer-stable]
+
+    name: PHP ${{ matrix.php }} - ${{ matrix.stability }}
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: ${{ matrix.php }}
+          extensions: dom, mbstring
+          tools: composer:v2
+          coverage: none
+
+      - name: Install dependencies
```

Comment:
Could we include composer validation step before installing dependencies?

```yml
    - name: Validate composer.json and composer.lock
      run: composer validate
```

Reply:
I've added that step in :)